Assets/Scripts/UI/InGame/HUD/KillInfo.cs

```diff
@@ -1,26 +1,33 @@
 ﻿using UnityEngine;
-using UnityEngine.UI;
+using TMPro;
 
 namespace Assets.Scripts.UI.InGame.HUD
 {
     public class KillInfo : UiElement
     {
-        private float alpha = 1f;
         private int col;
-        public Text KillerLabel;
-        public Text DamageLabel;
-        public Text VictimLabel;
+        public TMP_Text KillerLabel;
+        public TMP_Text DamageLabel;
+        public TMP_Text VictimLabel;
         private float lifeTime = 8f;
         private float maxScale = 1.5f;
         private int offset = 24;
         private bool start;
         private float timeElapsed;
 
+        [Header("Colors")] //Customize in the inspector
+        public Color fiveHundred;
```

Comment:
Rather use ScriptableObjects instead of hard coding this. Name it "DamageColor" or something similar. Within this ScriptableObject, including a Color, and a minimum damage number.

KillInfo will then contain a list of "DamageColors". The order of these DamageColors should be from low to high, then based on a quick loop, you can get the right DamageColor scriptableObject, and thus its color.